### PR TITLE
Remove header extension control negotiation restriction.

### LINKED ISSUE #60: Header extension control: spec text inconsistent with RFC5285 section 6.
> In the algorithm for generating subsequent offers in [RTCWEB-JSEP] section 5.2.2, replace "The RTP header extensions MUST only include those that are present in the most recent answer" with "For each RTP header extension listed in [[HeaderExtensionsToOffer]] that is also present in the most recent answer, and where direction is not "stopped", generate an appropriate "a=extmap" line with "direction" set according to the rules of [RFC5285] section 6, considering the direction in [[HeaderExtensionsToOffer]] to indicate the answerer's desired usage". This preserves the property that the set of extensions may not grow with subsequent offers."

But RFC5285, section 6 says:
> A session update MAY add or remove extension(s)

Given that extmap-allow-mixed is now turning default as of https://webrtc-review.googlesource.com/c/src/+/197943, there's soon no crisis anymore with the number of header extensions in use. I propose we drop the restriction.

### PR DESCRIPTION
**Pull request by handellm**
Fixes #60.

***
<a href="https://pr-preview.s3.amazonaws.com/handellm/webrtc-extensions/pull/62.html" title="Last updated on Jan 14, 2021, 3:44 PM UTC (15cbb68)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-extensions/62/d4a00f1...handellm:15cbb68.html" title="Last updated on Jan 14, 2021, 3:44 PM UTC (15cbb68)">Diff</a>

### CODE REVIEW

**Comment by alvestrand** [comment]
@dontcallmedom handellm is now a W3C and WG member. Where does he need to register the github<->registration association?

File: index.html [hunk 1]
```diff
@@ -270,16 +270,6 @@ <h2>Modifications to existing procedures</h2>
         line, as specified in [[RFC5285]], section 5, with direction taken from "e"'s direction
         attribute."
       </p>
-      <p>
-        In the algorithm for generating subsequent offers in [[RTCWEB-JSEP]] section 5.2.2, replace "The
-        RTP header extensions MUST only include those that are present in the most recent answer"
-        with "For each RTP header extension listed in [[\HeaderExtensionsToOffer]] that is
-        also present in the most recent answer, and where direction is not "stopped", generate
```
Comment (alvestrand) [inline index.html:277]: I think you just need to take out "also present in the most recent answer".

File: index.html [hunk 2]
```diff
@@ -270,16 +270,6 @@ <h2>Modifications to existing procedures</h2>
...
-        an appropriate "a=extmap" line with "direction" set according to the rules of [[RFC5285]]
-        section 6, considering the direction in [[\HeaderExtensionsToOffer]] to indicate the
-        answerer's desired usage". This preserves the property that the set of extensions
```
Comment (alvestrand) [inline index.html:280]: and that means the property is not preserved, so you have to delete "This preserves the property.... " too.

File: index.html (hunk 1 above)
Comment (handellm) [reply index.html:277]: Done.

File: index.html (hunk 2 above)
Comment (handellm) [reply index.html:280]: Done.

**Review by henbos — APPROVED** [review]

**Comment by henbos** [comment]
There are merge conflicts but you can merge it after rebasing

**Pull request merged**